test/files/run/complicatedmatch.scala

```diff
@@ -7,7 +7,7 @@ object Even{
 }
 
 object Test extends App{
-  val LongWord = "supercalifragilisticexpialadocious";
+  val LongWord = "supercalifragilisticexpialidocious";
```

Comment:
This one was unforgivable. Historical spellings include "Supercalafajalistickespeealadojus" and "Supercaliflawjalisticexpialidoshus". Srsly. https://en.wikipedia.org/wiki/Supercalifragilisticexpialidocious